packages/background/src/utils/constants/networks.ts

```diff
@@ -411,6 +411,37 @@ export const INITIAL_NETWORKS: Networks = {
         },
         nativelySupported: true,
     },
+    GNOSIS_RPCH: {
+        name: 'Gnosis_RPCh',
+        desc: 'Gnosis on RPCh',
+        chainId: 100,
+        networkVersion: '100',
+        nativeCurrency: {
+            name: 'xDAI',
+            symbol: 'xDAI',
+            decimals: 18,
+        },
+        hasFixedGasCost: false,
+        enable: true,
+        test: false,
+        order: 20,
+        features: [FEATURES.SENDS],
+        ens: false,
+        showGasLevels: false,
+        iconUrls: [
+            'https://raw.githubusercontent.com/block-wallet/assets/master/blockchains/xdai/assets/0x/logo.png',
+        ],
+        rpcUrls: ['https://primary.gnosis-chain.rpc.hoprtech.net'],
+        blockExplorerUrls: ['https://blockscout.com/xdai/mainnet'],
+        blockExplorerName: 'Blockscout',
+        etherscanApiUrl: 'https://api-gnosis.etherscan.io',
+        actionsTimeIntervals: { ...ACTIONS_TIME_INTERVALS_DEFAULT_VALUES },
```

Comment:
@diegoalzate changing this line will make transactions get confirmed quicker as it'll reduce the intervals that the extension waits to perform some actions like block pulling
```suggestion
        actionsTimeIntervals: { ...FAST_TIME_INTERVALS_DEFAULT_VALUES },
```